runner.go

```
@@ -219,6 +256,11 @@ func (r *Runner) Start() {
 			log.Printf("[INFO] (runner) received template %q from quiescence", tmpl.Path)
 			delete(r.quiescenceMap, tmpl.Path)
 
+		case c := <-childExitCh:
+			log.Printf("[INFO] (runner) child process died")
+			r.ErrCh <- fmt.Errorf("child process died with exit code %d", c)
```

Should we suppress error if `ExitCodeOK`?